components/TestimonialList/Patron.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import ImagePropTypes from '../../propTypes/image'
+import { rem } from '../../styling/typography'
+import { wordBreak } from '../../styling/utils'
+
+import RoundedImage from '../RoundedImage'
+
+const ImageContainer = styled.div`
+  max-width: 160px;
+`
+
+const Image = RoundedImage.extend`
+  margin-bottom: 0.7rem;
+`
+
+const Name = styled.div`
+  font-size: ${rem('18px')};
+  font-weight: bold;
+  line-height: 1.22;
+  color: ${({ theme }) => theme.orangeRed};
+  margin-bottom: 0.5rem;
+`
+
+const Title = styled.div`
+  font-size: ${rem('12px')};
+  line-height: 1.43;
+  color: ${({ theme }) => theme.pineCone};
+`
+
+const Patron = ({
```

Comment:
Shouldn't that be `Testimonial`?

Reply:
Now that I think more it is only the Image of the person with `name` and `title`.  I doesn't have anything to do with testimonial itself.
![screen shot 2018-02-13 at 17 33 40](https://user-images.githubusercontent.com/825836/36161554-47743102-10e4-11e8-994c-5c8a50fd855b.png)

Reply:
As discussed I will keep this name `Patron`